### DISCUSSION TITLE
None of the activites asigned to projects after import

### DISCUSSION THREAD

**Discussion by Dany1410** [kimai/kimai#3402] [Questions & Answers] [answered]
Hello everyone and Kevin,

I am trying to perform the upgrade from Kimai Version 1 to Kimai Version 2.
I got to manage to import my data from my current prod Database (DB) to my new kimai2 DB.
After the import I was able to login to the new web application Kimai2.
Unfortunality, all of the existing activities aren't asigned to the correspondig project as it used to be in my current Kimai1 Version 1.3.1.1394. 
=>[We use activities and projects as follows: There are several projects. To each project there is at least one activity asigned to it. Every activity is asigned to a project. There are no activities which aren't asigned to a project.]

In contrast, all of the projects are asigned correctly to the customers.
I am sure that I imported the whole prod DB incl. table 'kimai_projects_activities' which holds the information of which activity is asigned to which project.
My logfile does not give any warnings or errors related to the import of projects and/or activities.

Is this an import error? Malfunction of Kimai2?
Or am I interpreting a wrong philosophy of this application?

Best,
Dany

**Comment by kevinpapst** [comment, marked as answer]
You are using the `--global` flag, which creates global activities.
Kimai 2 has a slightly different look on activities compared to the old version. 
Please read the docs: https://www.kimai.org/documentation/activity.html
In short: you cannot link activities to multiple projects. They either exist for all or for exactly one project.

**Reply by Dany1410** [reply]
Yes, you are right.
Okay I see. I tried it without the global flag and it worked perfectly. 

But another case came up: When a user is permitted to select a project for time recording and wants to select a activity, all activites for this project are displayed. In Kimai Version 1 there was a predefined selection of activities the admin has chosen for a user to use. I know this feature is still there. But is it possible to migrate this information as well or do I have to adjust this afterwards manually? 

I had a look at your possible extensions for the import command but nothing really seems to fit my need and I did not use the flag --skip-team-activities.
Can you please help me @kevinpapst ? :)
